src/components/LogSimpleCard/index.tsx

```diff
@@ -0,0 +1,58 @@
+"use client";
+
+import { CSSProperties } from "react";
+import { LogSimpleCardContainer, LogSimpleCardBotton } from "./LogSimpleCard.style";
+
+interface LogSimpleCardProps {
+  width?: string;
+  height?: string;
+  radius?: string;
+  thumbnail: string;
+  location_1: string;
+  location_2: string;

```

Comment:
[ P4 ] 추후 Address 타입이 반영되면 Address 로 변경하신다면 좋을것 같습니다!
```suggestion

address : {
  depth1: string;
  depth2: string;
  depth3: string;
}
```